.github/workflows/CI.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         with:
           repository: 'opensearch-project/OpenSearch'
           path: OpenSearch
-          ref: 1.0.0-alpha2
+          ref: 1.0.0-beta1
 
       - name: Build OpenSearch
         working-directory: ./OpenSearch
```

Comment:
The qualifier should be updated to `beta1`. 
This will publish `alpha2` artifacts.

Reply:
Fixed it.